ee/maintained-apps/outputs/1password/windows.json

```diff
@@ -1,19 +1,19 @@
 {
   "versions": [
     {
-      "version": "8.10.64",
+      "version": "8.10.68",
       "queries": {
         "exists": "SELECT 1 FROM programs WHERE name = '1Password' AND publisher = 'AgileBits Inc.';"
       },
-      "installer_url": "https://downloads.1password.com/win/1PasswordSetup-8.10.64.msi",
+      "installer_url": "https://downloads.1password.com/win/1PasswordSetup-8.10.68.msi",
       "unique_identifier": "1Password",
       "install_script_ref": "8959087b",
       "uninstall_script_ref": "5910b9ba",
-      "sha256": "77479ffaef60167d2d0ba4cd85f3dd7914d248f417216d14cc9790a64abdeae4"
+      "sha256": "812a76a44145a63c5d1645fa9c9bbd5dd7e4e90b4efaccdb8f353c5be56688a6"
     }
   ],
   "refs": {
-    "5910b9ba": "$product_code = {DA39A6BE-3B6A-4D64-8B1D-EABFE806EF08}\n\n# Fleet uninstalls app using product code that's extracted on upload\nmsiexec /quiet /x $product_code\nExit $LASTEXITCODE\n",
+    "5910b9ba": "$product_code = {A51295E3-16C5-4A55-AA5A-BF1DEBCD69CA}\n\n# Fleet uninstalls app using product code that's extracted on upload\nmsiexec /quiet /x $product_code\nExit $LASTEXITCODE\n",
```

Comment:
@jahzielv The ref here didn't change. Is that expected? This is a different product code.

Reply:
Hmm yeah I would expect the ref to change. I can investigate.

Reply:
Found the issue. Will PR.